app/uk/gov/hmrc/economiccrimelevyaccount/services/FinancialDataService.scala

```diff
@@ -125,7 +124,26 @@ class FinancialDataService @Inject() (
           }
       }
 
-    FinancialViewDetails(outstandingPayments = outstandingPayments, paymentHistory = paymentsHistory)
+    val accruingInterestOutstandingPayments = documentDetails
+      .collect(filterItemsThatHaveAccruingInterest)
+      .filter(document => document.interestAccruingAmount.nonEmpty)
```

Comment:
Isn't this redundant if **filterItemsThatHaveAccruingInterest** already has **filterItemsThatHaveAccruingInterestAmount**?

Reply:
Oh, correct correct, left it in during the refactoring. Thank you, it's removed now.